src/index.d.ts

```diff
@@ -12,6 +12,7 @@ export interface ExplosionProps {
   fadeOut?: boolean;
   autoStart?: boolean;
   autoStartDelay?: number;
+  renderToHardwareTextureAndroid?: boolean;
```

Comment:
i'm not sure we have to make it customizable if it creates UI glitches ?
What do you think about setting it to true everytime and making it transparent for developers ?

Reply:
Yep great point!  My original thought was to allow users to opt-out of the feature, but your point about Android being unusable anyways without it completely negates that.

Reverted all the changes made to expose this externally 👍 .